docs/README.md

```diff
@@ -1,9 +1,67 @@
 # Documentation
+
 ## Overview
-This directory contains the following documents that relate to the project:
 
-- [Installation](https://github.com/kyma-project/helm-broker/blob/master/docs/installation.md) describes how to install the Helm Broker.
-- [Configuration](https://github.com/kyma-project/helm-broker/blob/master/docs/configuration.md) describes the environment variables you can configure.
-- [Development](https://github.com/kyma-project/helm-broker/blob/master/docs/development.md) describes how to develop the project.
-- [Releasing](https://github.com/kyma-project/helm-broker/blob/master/docs/releasing.md) describes the Helm Broker release process.
-- [Example usage](https://github.com/kyma-project/helm-broker/blob/master/docs/example-usage.md) describes how to provision a Redis instance using the Helm Broker and addons.
+This directory contains Helm Broker documentation.
+
+The Helm Broker fetches addons which contain a set of specific [files](#details-create-addons). You must place your addons in a repository of an appropriate [format](#details-create-addons-repository). The Helm Broker fetches default cluster-wide addons defined by the [helm-repos-urls](https://github.com/kyma-project/kyma/blob/master/resources/helm-broker/templates/default-addons-cfg.yaml) custom resource (CR). This CR contains URLs that point to the release of [`addons`](https://github.com/kyma-project/addons/releases) repository compatible with a given [Kyma release](https://github.com/kyma-project/kyma/releases). You can also configure the Helm Broker to fetch addons definitions from other addons repositories.
+
```

Comment:
we can add here a sentence like: look here's the documentation about installation and development